charts/karpenter/README.md

```diff
@@ -3,7 +3,7 @@
 A Helm chart for Kubernetes that provides an implementation of Karpenter using Cluster API as the infrastructure
 provider
 
-![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.2.0](https://img.shields.io/badge/AppVersion-0.2.0-informational?style=flat-square)
```

Comment:
Just trying to understand how we are versioning this project, are we doing chart releases separate?